.github/workflows/tests.yml

```diff
@@ -206,3 +206,31 @@ jobs:
         v=${GITHUB_REF:10}
         docker tag brew "docker.pkg.github.com/homebrew/brew/brew:$v"
         docker push "docker.pkg.github.com/homebrew/brew/brew:$v"
+
+  docker:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        tag: [ubuntu18, ubuntu20]
+    steps:
+    - name: Checkout
+      uses: actions/checkout@master
+      with:
+        fetch-depth: 0
+    - name: Build Docker image
+      run: docker build -t brew -f docker/${{matrix.tag}}/Dockerfile .
+    - name: Run brew test-bot
+      run: docker run --rm brew brew test-bot
+    - name: Deploy the latest Docker image
```

Comment:
Is this to DockerHub, GitHub, both or neither?

Reply:
In this PR the image is being deployed to GitHub Packages. In a follow-up PR, I plan to deploy to both Docker Hub and GitHub Packages, so that we can wind down the automated builds on `hub.docker.com`.

Reply:
could you clarify in the description?

Reply:
Done.